.changeset/encoding-static-builds.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': major
+---
+
+Routes with `%25` (percent-encoded percent sign) no longer supported
```

Comment:
Just a note that we've had users raising issues about encoding, so maybe we should add this to the docs too